project-base/app/src/Model/Transfer/Issue/TransferIssue.php

```diff
@@ -13,7 +13,7 @@
  * @ORM\Table(
  *     name="transfer_issues",
  *     indexes={
- *          @ORM\Index(columns={"created_at"}),
+ *          @ORM\Index(columns={"created_at", "deleted_at"}),
```

Comment:
if the index is here primarily because of the following query, we should add `transfer_id` to the index as well :thinking: 

```
return $this->getQueryBuilder()
            ->select('ti, t')
            ->join(Transfer::class, 't', Join::WITH, 'ti.transfer = t')
            ->where('ti.deletedAt IS NULL')
            ->andWhere('ti.createdAt > :fromDateTime')
            ->setParameter('fromDateTime', $fromDateTime)
            ->orderBy('ti.createdAt', 'DESC')
            ->addOrderBy('ti.id', 'DESC');
```

Reply:
index for `transfer_id` already exists, I don't think it's really necessary to create composite index for every usage of column, I agree it can be better but I don't think that will speed up extremely but if you want it I can add it to index

Reply:
yeah, add it there, please, you can send it to BV then :wink:

Reply:
ok, i added `transfer_id` to index and migration and send it to BV, thanks 😊